test/edu/ucsb/nceas/metacat/startup/MetacatInitializerIT.java

```diff
@@ -0,0 +1,152 @@
+package edu.ucsb.nceas.metacat.startup;
+
+import java.io.IOException;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+
+import org.dataone.configuration.Settings;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import org.mockito.Mockito;
+
+import edu.ucsb.nceas.LeanTestUtils;
+import edu.ucsb.nceas.metacat.index.queue.IndexGenerator;
+import edu.ucsb.nceas.metacat.properties.PropertyService;
+import edu.ucsb.nceas.metacat.util.SystemUtil;
+import edu.ucsb.nceas.utilities.PropertyNotFoundException;
+
+/**
+ * The IT test class for the MetacatInitializer class
+ * @author tao
+ *
+ */
+public class MetacatInitializerIT {
+    private ServletContextEvent event = null;
+
+    @Before
+    public void setUp() throws Exception {
+        LeanTestUtils.initializePropertyService(LeanTestUtils.PropertiesMode.LIVE_TEST);
```

Comment:
AFAICT, there's nothing in this test that requires metacat to be running. It therefore should be using PropertiesMode.UNIT_TEST. You're already mocking the `ServletContextEvent` - which is great. It looks like the only reason you are using the LIVE mode is to change the "configured" Properties for different tests. This can all be done with UNIT test mode.

(I can help figure out how to do this if you want)

Reply:
This test needs Postgresql is running and configure, needs rabbitmq is running and et al. So it wouldn't work of the box. So I call it `IT`.